src/popover.test.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import Popover from './Popover';
+import PopoverTrigger from './PopoverTrigger';
+import PopoverMenu from './PopoverMenu';
+import PopoverMenuItem from './PopoverMenuItem';
+
+describe('Popover', () => {
+	const class_menu = 'popover-container',
+		class_trigger = 'popover-trigger',
+		class_option = 'popover-menu-item',
+		class_hidden = 'display--none';
+
+	let popover,
+		popoverEl,
+		triggerEl,
+		menuEl,
+		optionEls;
+
+	beforeEach(() => {
+		popover = TestUtils.renderIntoDocument(
+			<Popover>
+				<PopoverTrigger>Trigger</PopoverTrigger>
+				<PopoverMenu>
+					<PopoverMenuItem>One</PopoverMenuItem>
+					<PopoverMenuItem>Two</PopoverMenuItem>
+					<PopoverMenuItem>Three</PopoverMenuItem>
+				</PopoverMenu>
+			</Popover>
+		);
+		popoverEl = ReactDOM.findDOMNode(popover);
+		triggerEl = TestUtils.scryRenderedDOMComponentsWithClass(class_trigger)[0];
+		menuEl = TestUtils.scryRenderedDOMComponentsWithClass(class_menu)[0];
+		optionEls = TestUtils.scryRenderedDOMComponentsWithClass(class_option);
+	});
+
+	afterEach(() => {
+		popover = null;
+		popoverEl = null;
+		triggerEl = null;
+		menuEl = null;
+		optionEls = null;
+	});
+
+	it('exists; menu hidden by default', () => {
+		console.warn(popoverEl);
+		expect(popoverEl).not.toBeNull();
+		expect(menuEl.classList.contains(class_hidden)).toBe(true);
+	});
+
+	it('menu appears on trigger click', () => {
+		TestUtils.Simulate.click(triggerEl);
```

Comment:
for user interaction tests, it's good practice to test the state both before and after the simulated interaction so that you can be sure that something changed. It's not quite as big a deal here because the previous unit test tests the 'before' state, but it's also best practice to keep unit tests completely independent